packages/samples/headless-react/src/pages/AtomicReactPage.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+  AtomicResultList,
+  AtomicSearchBox,
+  AtomicSearchInterface,
+} from '@coveo/atomic-react';
+import {useEffect, useRef} from 'react';
+
+export function AtomicReactPage() {
+  return (
+    <AtomicSearchInterfaceWrapper>
+      <AtomicSearchBox />
+      <AtomicResultList />
+    </AtomicSearchInterfaceWrapper>
+  );
+}
+
+interface WrapperProps {
+  onReady?: () => void;
+}
+
+function AtomicSearchInterfaceWrapper(
```

Comment:
I extracted this wrapper following the comments. However, if the wrapper is imported from `@coveo/atomic-react`, then [this project errors](https://github.com/coveo/ui-kit/pull/1600#discussion_r770809329). I haven't been able to figure out why so far,